BUILD

```diff
@@ -353,7 +353,7 @@ cc_test(
     name = "distbench_test_sequencer_test",
     size = "medium",
     srcs = ["distbench_test_sequencer_test.cc"],
-    shard_count = 8,
+    shard_count = 26,
```

Comment:
I wonder if it isn't too much for Kokoro (having all the tests running at the same time).

Reply:
bazel should still limit the number of running tests to # CPUs.